package.json

```diff
@@ -1,25 +1,24 @@
 {
+  "name": "tldr-pages",
   "description": "Simplified and community-driven man pages",
   "author": "Romain Prieto",
   "license": "SEE LICENSE IN LICENSE.md",
   "repository": "tldr-pages/tldr",
   "homepage": "https://tldr.sh/",
   "dependencies": {
-    "glob": "7.1.3",
-    "husky": "^2.2.0",
-    "markdownlint-cli": "0.19.0",
+    "glob": "7.1.6",
+    "markdownlint-cli": "0.27.1",
     "tldr-lint": "~0.0.10"
   },
   "scripts": {
     "lint-markdown": "markdownlint pages*/**/*.md",
     "lint-tldr-pages": "tldr-lint ./pages",
     "test": "bash scripts/test.sh",
-    "build-index": "node ./scripts/build-index.js > index.json"
+    "build-index": "node ./scripts/build-index.js > index.json",
+    "postinstall": "husky install"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm test"
-    }
-  },
-  "private": true
+  "private": true,
+  "devDependencies": {
+    "husky": "^5.1.3"
+  }
```

Comment:
I like the `devDeps` under the prod `dependencies`, and the `scripts` above `dependencies`. But this `package.json` has a rare order lol.

Reply:
No problem, I'll do that.  As a matter of fact though, for some reason it's common to see `devDependencies` at the bottom ¯\\_(ツ)_👍🏻

Reply:
I really like the idea of an "official" `package.json` linter/prettier.
Is there something more official than this? https://github.com/cameronhunter/prettier-package-json/blob/HEAD/src/defaultOptions.js

Reply:
https://npmpackagejsonlint.org/ || https://github.com/tclindner/npm-package-json-lint?

Reply:
@bl-ue I think that's because npm adds new properties to the bottom of `package.json`, and generally speaking `devDependencies` is the last thing that most developers add via an `npm install --save-dev {{package_name}}`